front-end/src/components/Login.jsx

```diff
@@ -0,0 +1,174 @@
+import React, {useEffect, useState} from 'react'
+import {Formik, Form, Field, ErrorMessage} from 'formik'
+import * as Yup from 'yup'
+import { Link } from 'react-router-dom'
+import { useHistory } from 'react-router-dom'
+import Footer from './Footer'
+import axios from "axios";
+import { Alert, Container } from 'react-bootstrap'
+import Header from './Header'
+
+const Login = (props) => {
+const {history} = props
+
+    const [loadginLogin, setLoadingLogin] = useState(false);
+    const [errorLogin, setErrorLogin] = useState(false);
+    const [successLogin, setSuccessLogin] = useState(false);
+    const [email, setEmail] = useState('');
+    const [token, setToken] = useState('');
+
+
+    function generateToken () {
+        const token = Math.random().toString(36).substring(2, 15) + Math.random().toString(36).substring(2, 15);
+        setToken(token);
+    }
+    useEffect(() => {
+        generateToken();
+    } , [token]);
+
+        const handleClickLogin = (values) => { 
+      axios.post("http://localhost:3001/login", {
+      email: values.email,
+      password: values.password,
+    }).then((response) => {
+      if (response.data === "Login realizado com sucesso") {
```

Comment:
Se o response for a string contida no data então logar.
Ficaria melhor se o response retornasse um status e uma mensage.